src/controllers/Product.controller.ts

```diff
@@ -81,3 +84,62 @@ export const getAvailableProducts = async (req: Request, res: Response) => {
     }
   }
 };
+
+// Seller getting all items
+export const getAllSellerItems = async (req: Request, res: Response) => {
+  try {
+    const allItems = await getAllItemsService();
+    return res.status(200).json({
+      statusCode: 200,
+      success: true,
+      data: allItems,
+    });
+  } catch (error) {
+    if (error instanceof Error) {
+      res.status(500).json({
+        status: 500,
+        success: false,
+        message: 'Something went wrong when getting the products',
+        error: error.message,
+      });
+    } else {
+      console.log(`Unexpected error: ${error}`);
+    }
+  }
+};
+
+// Seller update a product
+export const updateProduct = async (req: Request, res: Response) => {
+  try {
+    const productId = req.params.id;
+    const productToUpdate = await findProductService(productId);
+    const parsedDataToUpdate = JSON.parse(JSON.stringify(productToUpdate));
+
+    for (const productCriteria in req.body) {
+      if (req.body[productCriteria]) {
+        parsedDataToUpdate[productCriteria] = req.body[productCriteria];
+      }
+    }
+    const updatedProduct = await updateProductService(
+      productId,
+      parsedDataToUpdate
+    );
+
+    return res.status(200).json({
+      statusCode: 200,
```

Comment:
These response messages thought not fundamentally different, they are structurally different and could lead to confusion down the road. 
Check other instances in the code-base and use the same format we are using.